src/dev/flang/tools/AceMode.java

```diff
@@ -0,0 +1,250 @@
+/*
+
+This file is part of the Fuzion language implementation.
+
+The Fuzion language implementation is free software: you can redistribute it
+and/or modify it under the terms of the GNU General Public License as published
+by the Free Software Foundation, version 3 of the License.
+
+The Fuzion language implementation is distributed in the hope that it will be
+useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+License for more details.
+
+You should have received a copy of the GNU General Public License along with The
+Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+*/
+
+/*-----------------------------------------------------------------------
+ *
+ * Tokiwa Software GmbH, Germany
+ *
+ * Source of class AceMode
+ *
+ *---------------------------------------------------------------------*/
+
+package dev.flang.tools;
+
+import dev.flang.parser.Lexer;
+import dev.flang.util.ANY;
+import java.util.stream.*;
```

Comment:
I keep imports sorted alphabetically and packages separated by blank lines, i.e., 
```suggestion
import dev.flang.parser.Lexer;

import dev.flang.util.ANY;

import java.util.stream.Stream;
```
even though in this case this does not make a bit difference since there are only single classes imported from each package.

Reply:
Done in #88d9dff